.env

```diff
@@ -26,11 +26,6 @@ APP_SECRET=9947170bb921f0390a44d613fa7a3ce5
 # DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7"
 DATABASE_URL="postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=13&charset=utf8"
 ###< doctrine/doctrine-bundle ###
-
-###> sentry/sentry-symfony ###
-SENTRY_DSN=
```

Comment:
Geen sentry meer?

Reply:
Ik wou zo snel mogelijk upgraden naar 6.1, en sentry/symfony-sentry deed moeilijk. Leek mij ook niet echt een meerwaarde, aangezien dit toch geen echt project is maar eerder een voorbeeld repo. We developen er enkel lokaal op en er zit al phpstan/phpunit/phpcs in.

Reply:
true, agreed.